themes/bootstrap3/js/check_item_statuses.js

```diff
@@ -41,17 +41,15 @@ VuFind.register('itemStatuses', function ItemStatuses() {
       el.querySelectorAll('.callnumber,.hideIfDetailed,.location').forEach((e) => e.classList.add('hidden'));
       el.querySelectorAll('.locationDetails').forEach((locationDetails) => {
         locationDetails.classList.remove('hidden');
-        locationDetails.innerHTML = result.locationList;
+        VuFind.setInnerHtml(locationDetails, result.locationList);
       });
     } else {
       // Default case -- load call number and location into appropriate containers:
       el.querySelectorAll('.callnumber').forEach((callnumber) => {
-        callnumber.innerHTML = typeof(result.callnumberHtml) !== 'undefined' && result.callnumberHtml
-          ? result.callnumberHtml + '<br>'
```

Comment:
I see that you're dropping the `<br>` here, which is probably a good thing. Just double-checking whether you confirmed that doing so has no visible side-effects.

Reply:
Ill double check this one